sys/fuchsia/process.txt

```diff
@@ -0,0 +1,22 @@
+# Copyright 2022 syzkaller project authors. All rights reserved.
+# Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+# See: https://cs.opensource.google/fuchsia/fuchsia/+/main:zircon/vdso/process.fidl
+
+include <zircon/syscalls.h>
+
+resource zx_process[zx_handle]
+
+zx_process_exit(ret_code int64)
```

Comment:
`ignore_return` is needed as it doesn't return a status.
But fuzzing-wise, will it break execution ? In this case, add `break_returns` too - or maybe even `disabled` ?

Reply:
> But fuzzing-wise, will it break execution ?

It doesn't seem to.
If you click on "coverage" in syz-manager interface, it shows all return values it collected from all syscalls and on syzbot I see an empty set for zx_process_exit.
`break_returns` is intended for calls that mess with return values of _subsequent_ syscalls. E.g. on Linux syzkaller managed to make exit_group syscalls fail by installing a seccomp filter that prohibits exit_group and instead returns random errors from it. So IIRC we should marked seccomp as `break_returns`.

Reply:
Added `ignore_return`.